### PR TITLE
Jessica/13256 include root exception

### PR DESCRIPTION
**Pull request by JessicaWNava**
This PR adds the exception to all schema exceptions that have an exception available and were not previously passing it along. 

Test Steps:
1. Run the unit tests

## Changes
- adds the exception to all schema exceptions that have an exception available and were not previously passing it along. 

### Testing
- [ ] Tested locally?
- [x] Ran `./prime test` or `./gradlew testSmoke` against local Docker ReportStream container?
- [ ] (For Changes to /frontend-react/...) Ran `npm run lint:write`? 
- [x] Added tests?

## Linked Issues
- #issue 13256

### CODE REVIEW

File: prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/FhirPathUtilsTests.kt [hunk 1]
```diff
@@ -73,6 +73,20 @@ class FhirPathUtilsTests {
...
+    fun `test evaluate condition error thrown`() {
+        val bundle = Bundle()
+        bundle.id = "abc123"
+
+        val path = "Bundle.entry[0].resource.blah('blah')"
+        try {
+            FhirPathUtils.evaluateCondition(null, bundle, bundle, bundle, path)
+        } catch (e: Exception) {
+            assertThat(e).isInstanceOf<SchemaException>()
+            assertThat(e.cause!!.message).isEqualTo("Error in ?? at 1, 1: The name blah is not a valid function name")
```
Comment (jalbinson) [inline prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/FhirPathUtilsTests.kt:86]: You can chain assertions to have better output if they fail and update the type you are working with.

```kotlin
assertThat(e.cause). // nullable exception
  .isNotNull() // non-nullable exception
  .isInstanceOf<UnderlyingExceptionType>() // checking the type matches what we expect
```

**Review by thetaurean — APPROVED** [review]
Looks good 👍  Great job scanning through all the usages of SchemaException and not missing any changes!

I have one small comment with a small suggestion; happy to approve otherwise.

File: prime-router/settings/organizations.yml [hunk 2]
```diff
@@ -93,6 +93,8 @@
         - "(%performerState.exists() and %performerState = 'IG') or (%patientState.exists() and %patientState = 'IG')"
       qualityFilter:
         - "(%hl7MessageType.exists() and %hl7MessageType = 'R01')"
+      conditionFilter:
+        - "resource.livdTableLookup('Component').contains('blahblahblah').not()"
```
[the remark this answers is not in the document]
Comment (JessicaWNava) [reply prime-router/settings/organizations.yml:97]: Oh, sorry! I actually totally meant to remove that. Fixing.

File: prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/FhirPathUtilsTests.kt (hunk 1 above)
Comment (JessicaWNava) [reply prime-router/src/test/kotlin/fhirengine/translation/hl7/utils/FhirPathUtilsTests.kt]: Cool idea, thanks!

**Review by thetaurean — APPROVED** [review]
👍

**Review by jalbinson — APPROVED** [review]

**Review by arnejduranovic — APPROVED** [review]

**Pull request merged**